cumulus_library/template_sql/sql_utils.py

```diff
@@ -258,19 +258,17 @@ def denormalize_complex_objects(
 
 def validate_schema(
     database: databases.DatabaseBackend,
-    expected_table_cols: dict,
+    expected_table_cols: dict[dict],
```

Comment:
I'm used to dict[key, value] - what's this typing mean?

Reply:
nothing - will update to `dict[key,dict[key,value]]`